addons/api/addon/models/worker.js

```diff
@@ -31,14 +31,24 @@ export default class WorkerModel extends GeneratedWorkerModel {
    * @type {number}
    */
   get tagCount() {
-    if (!this.config_tags) {
+    if (!this.config_tags && !this.api_tags) {
```

Comment:
Same here, should we just have been looking at `canonical_tags` instead rather than adding them up ourselves?

Reply:
For sure here we could use `canonical_tags`. I will update.